package.json

```
@@ -20,21 +20,21 @@
   "license": "MIT",
   "dependencies": {
     "@blockr/blockr-logger": "^1.3.0",
-    "@blockr/blockr-models": "^1.4.0-build.6",
-    "@types/jest": "^24.0.13",
+    "@blockr/blockr-models": "^1.4.0-build.12",
```

Latest build is 15.
```suggestion
    "@blockr/blockr-models": "^1.4.0-build.15",
```
After run `npm i`